llvm/CMakeLists.txt

```diff
@@ -673,6 +673,9 @@ option(LLVM_USE_OPROFILE
 option(LLVM_EXTERNALIZE_DEBUGINFO
   "Generate dSYM files and strip executables and libraries (Darwin Only)" OFF)
 
+option(LLVM_ENABLE_NO_EXPORTED_SYMBOLS
```

Comment:
The double negative is confusing. Maybe invert it, and make the name `LLVM_ENABLE_EXPORTED_SYMBOLS`?

Reply:
should the name reflect the opposite of the default? if so maybe `LLVM_DISABLE_EXPORTED_SYMBOLS`?

Reply:
No preference here. Whatever seems easier for others to understand.

Reply:
Flags like this shouldn't have `DISABLE` in their name either. It's ambiguous/confusing whether `LLVM_DISABLE_THING=On` means THING=ON or THING=Off.